django_rest_passwordreset/models.py

```diff
@@ -63,8 +63,8 @@ def generate_key():
         blank=True,
         null=True,
     )
-    user_agent = models.CharField(
-        max_length=256,
+    user_agent = models.TextField(
```

Comment:
This should remain a `CharField`

Reply:
Yep, i'm already fix it, in last commits

Reply:
@mvasilchenko Can you correct `TextField` in your PR? :)